### ISSUE TITLE
-unixpw / -unixpw_nis does not work on Alpine Linux

### ISSUE THREAD

**Issue by dustyhorizon** [LibVNC/x11vnc#37]
Hello,

I compiled libvncserver and x11vnc through git sources without any errors and managed to launch x11vnc. However, using -unixpw fails everytime with the logs showing

```
x11vnc stderr | unixpw_verify: su_verify login for 'user' failed.
x11vnc stderr | 02/02/2017 04:18:14 unixpw_verify: 'user'
x11vnc stderr | 02/02/2017 04:18:14 su_verify: 'user' for login.
x11vnc stderr | 02/02/2017 04:18:14 unixpw_verify: su_verify login for 'user' failed.
```

There is no prior error before that however there is one line with 'unixpw' when I first connect.

```
x11vnc stderr | 02/02/2017 04:17:50 new client: ::1 in non-unixpw_in_rfbPE.
x11vnc stderr | 02/02/2017 04:17:50 rfbPE: got new client in non-rfbPE
```

I connect via a SSH local tunnel via VNC Viewer client from UltraVNC.

As with the error with -unixpw_nis, it says that this os/machine is unsupported.

Do let me know if further logs / info is needed

**Comment by bk138** [comment]
There have been fixes to this lately (1ae1ef1ca6173081913325a682fac4cca1a61321), pls try again with this version!

**Comment by bk138** [comment]
Closing as this mose likely got fixed with https://github.com/LibVNC/x11vnc/commit/1ae1ef1ca6173081913325a682fac4cca1a61321. Feel free to re-open if issue persists.
